Sources/EdgeNetworkHandlers/EdgeHitProcessor.swift

```diff
@@ -224,4 +229,15 @@ class EdgeHitProcessor: HitProcessing {
         return requestHeaders
     }
 
+    private func getCustomRequestPath(from event: Event) -> String? {
+        var customPath: String?
+        if let eventData = event.data {
+            let xdmData = eventData[EdgeConstants.JsonKeys.XDM] as? [String: Any]
+            let requestData = xdmData?[EdgeConstants.JsonKeys.REQUEST] as? [String: Any]
+            customPath = requestData?[EdgeConstants.JsonKeys.PATH] as? String
```

Comment:
Usually a path starts with slash, so I am thinking we should require that like `/va/v1/sessionStart`. We could also verify that it does not start multiple slashes (// is used before the domain/authority component) and if so ignore and log a debug message, though can you verify how will path.path.append behave in that case e.g. //va/v1?